packages/nimble/R/RCfunction_compile.R

```diff
@@ -265,10 +265,15 @@ RCfunProcessing <- setRefClass('RCfunProcessing',
                                            browser()
                                        }
 
+                                       if(nimbleOptions('useRefactoredSizeProcessing')) {
```

Comment:
It's a good habit to name these short-lived flags `experimentalMyFlagName` or `internalMyFlagName` to make it clear to our users that they shouldn't use these flags.